variables.tf

```diff
@@ -78,12 +78,24 @@ variable "tags" {
   default     = {}
 }
 
+variable "policy_arn" {
+  description = "An addional policy (ARN) to attach to the Lambda function"
+  type        = "string"
+  default     = ""
+}
+
 variable "policy" {
   description = "An addional policy to attach to the Lambda function"
   type        = "string"
   default     = ""
 }
 
+variable "attach_policy_arn" {
```

Comment:
This variable won't be needed if `policy_arns` is a list.

Reply:
Replaced by _count_